torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -875,6 +890,7 @@ def benchmark_all_configs(self, *args, **kwargs):
                         k.shared,
                     )
 
+            self.reset_to_zero_args(*args, **kwargs)
```

Comment:
Do we add this because of this line in Triton? https://github.com/triton-lang/triton/blob/aaf64d68f6da0593dbc96d9ba998c10af540ce4f/python/triton/runtime/autotuner.py#L194

Reply:
I think we have to do this because reset_to_zero executes inside pre_hook: https://github.com/triton-lang/triton/blob/aaf64d68f6da0593dbc96d9ba998c10af540ce4f/python/triton/runtime/autotuner.py#L64 


So pre_hook runs before every run, but also after the benchmarking is complete, this matches the behavior (I think)

This is the case unless "self.user_defined_pre_hook = True", in which case reset_to_zero doesn't run